docs/source/share_kedro_viz.md

```diff
@@ -109,3 +109,17 @@ You can control who can view your visualisation using [bucket and user policies]
 You pay for storing objects in your S3 buckets. The amount you pay depends on your objects’ size, how long you stored the object during the month, and the storage class.
 
 See the official [AWS documentation](https://aws.amazon.com/s3/pricing/?nc=sn&loc=4) for more information. 
+
+## Platform-agnostic sharing with Kedro-Viz 
+
+In Kedro-Viz version 7.1.0, we introduced the `kedro viz build` command that enables you to publish and share Kedro-Viz to any static website hosting platform. Running this command from the command line interface (CLI) creates a `build` folder within the Kedro project. The build folder contains a static Kedro-Viz app package, which can be used as a source to publish on any static website hosting platform.
+
+### Static website hosting platforms such as GitHub Pages
+
+Follow the steps [listed in the GitHub pages documentation](https://docs.github.com/en/pages/quickstart) to create a Git repository that supports GitHub Pages. On completion, push the contents of the `build` folder to this new repository. Your site will be available at the following URL: `http://<username>.github.io`
+
+### Cloud providers such as Azure, and GCP
+
+After creating a bucket and configuring it for static website hosting, copy the contents of the `build` folder to the bucket manually. You can then access the bucket via its endpoint.
+
+For AWS, we offer deploy orchestration to [automate the upload process as described above](#publish-and-share-kedro-viz-automatically).
```

Comment:
Can we not use the word "orchestration" here to avoid confusion? Maybe "an automatic process"

Reply:
```suggestion
For AWS, we offer [automated deployment as described above](#publish-and-share-kedro-viz-automatically).
```